src/control/drpc/property.go

```diff
@@ -8,8 +8,11 @@ package drpc
 import "unsafe"
 
 /*
+#cgo LDFLAGS: -ldaos_common -lcart -lgurt
```

Comment:
same question, why cart lib is required?

Reply:
removed

Reply:
Remove it caused following build failures:

# github.com/daos-stack/daos/src/control/drpc
/tmp/go-build1521650367/b275/_x003.o: In function `_cgo_ab7930e417a8_Cfunc_daos_policy_try_parse':
property.cgo2.c:(.text+0x22e): undefined reference to `daos_policy_try_parse'
collect2: error: ld returned 1 exit status
# github.com/daos-stack/daos/src/control/drpc
/tmp/go-build794554783/b272/_x003.o: In function `_cgo_ab7930e417a8_Cfunc_daos_policy_try_parse':
property.cgo2.c:(.text+0x22e): undefined reference to `daos_policy_try_parse'
collect2: error: ld returned 1 exit status
# github.com/daos-stack/daos/src/control/drpc
/tmp/go-build2304956098/b272/_x003.o: In function `_cgo_ab7930e417a8_Cfunc_daos_policy_try_parse':
property.cgo2.c:(.text+0x22e): undefined reference to `daos_policy_try_parse'

I tried to add it back.